### ISSUE TITLE
Render error on Windows

### ISSUE THREAD

**Issue by jeremiedb** [RePsychLing/SMLP2022#1]
I tried having a preview of the project to better understand Quarto. 
Following a clone of the project, from a CLI, I entered: `quarto render`, which failed with the following error: 

```bash
[1/8] index.qmd
[2/8] about.qmd
[3/8] notebooks\useful_packages.qmd

Starting Jupyter kernel...Done


An error occurred while executing the following cell:
------------------
import IJulia

# The julia kernel has built in support for Revise.jl, so this is the
# recommended approach for long-running sessions:
# https://github.com/JuliaLang/IJulia.jl/blob/9b10fa9b879574bbf720f5285029e07758e50a5e/src/kernel.jl#L46-L51

# Users should enable revise within .julia/config/startup_ijulia.jl:
# https://timholy.github.io/Revise.jl/stable/config/#Using-Revise-automatically-within-Jupyter/IJulia-1

# clear console history
IJulia.clear_history()

# Intialize Plots w/ default fig width/height
try
  fig_width = 7
  fig_height = 5
  fig_format = :retina
  fig_dpi = 96
  # no retina format type, use svg for high quality type/marks
  if fig_format == :retina
    fig_format = :svg
  # IJulia doesn't support PDF output so use png (if the DPI
  # remains the default of 300 then set to 96)
  elseif fig_format == :pdf
    fig_format = :png
    fig_dpi = 96
  end
  # convert inches to pixels
  fig_width = fig_width * fig_dpi
  fig_height = fig_height * fig_dpi
  using Plots
  gr(size=(fig_width, fig_height), fmt = fig_format, dpi = fig_dpi)
catch e
  # @warn "Plots init" exception=(e, catch_backtrace())
end

# Set run_path if specified
try
  run_path = "C:\Users\jerem\OneDrive\JuliaComputing\Quarto\SMLP2022"
  if !isempty(run_path)
    cd(run_path)
  end
catch e
  @warn "Run path init:" exception=(e, catch_backtrace())
end

# don't return kernel dependencies (b/c Revise should take care of dependencies)
nothing

------------------

syntax: invalid escape sequence

Stacktrace:
 [1] top-level scope
   @ In[1]:39
 [2] eval
   @ .\boot.jl:373 [inlined]
 [3] include_string(mapexpr::typeof(REPL.softscope), mod::Module, code::String, filename::String)
   @ Base .\loading.jl:1196
LoadError: syntax: invalid escape sequence
```

**Comment by palday** [comment]
Thanks for trying things out. :smile:

I'm honestly at a bit of a loss for what's causing that error -- we never actually load `IJulia` in that notebook, but I believe Quarto does that for interfacing with Jupyter. I don't have a Windows machine handy at the moment, but my strong suspicion is that it's your path that's causing problems:
```julia
run_path = "C:\Users\jerem\OneDrive\JuliaComputing\Quarto\SMLP2022"
```

I believe that's a Quarto bug -- Quarto doesn't escape `\` and/or use a `raw` string, so then you have an invalid escape sequence.

**Comment by jeremiedb** [comment]
Hello, I just gave another try, this time through Ubuntu 20.05 / WSL2, and hit another issue: 
```
Error running filter /opt/quarto/share/filters/quarto-post/quarto-post.lua:
/opt/quarto/share/filters/quarto-post/quarto-post.lua:1752: attempt to index a nil value (field 'attr')
stack traceback:
        /opt/quarto/share/filters/quarto-post/quarto-post.lua:1063: in function </opt/quarto/share/filters/quarto-post/quarto-post.lua:1057>
```

Quarto version: `0.9.531`.

Error looks similar to https://github.com/quarto-dev/quarto-cli/issues/21, although the patch should have been integrated for a while. Any hint what might going wrong? Are you aware if anyone was successful building the website from WSL2?

**Comment by palday** [comment]
@jeremiedb did you install Quarto through the dpkg available on the Quarto website?

**Comment by jeremiedb** [comment]
Yes, package was installed with the Linux dpkg. However, I noticed that new quarto versions recently got released and the website build successfully with version `0.9.594`. Not sure what was fix between `0.9.531` and this one, but it now works! Thanks, looking sharp!
